src/Artesian/GMEPublicOffers/_Enum/Market.py

```diff
@@ -18,3 +18,7 @@ class Market(Enum):
     MI7 = 14
     MSD = 15
     MRR = 16
+    MIA1 = 17
```

Comment:
Enum values are not aligned with Service/C#SDK ([reference](https://github.com/ARKlab/Artesian.SDK/blob/76bd5de3736647d34895f7065ff614a90e3fb932/Artesian/Artesian.SDK/Dto/GMEPublicOffer/Enum/GME_Enums.cs#LL4C6-L4C6)). this is something that was already present before this PR.

is there a reason for not aligning them?

Reply:
change in enumeration requested by @MicheleNan following initial review request in which order aligned with Service/C#SDK

Reply:
No specific reason, as for align them to the service order, in the end is just an ENUM. 
If @AndreaCuneo prefer to have them aligned, revert the change, although I'm not sure if it Python is suggested to start an enum from 0 ( as we're doing on the service )

Reply:
In absense of a specific reason, I prefer them aligned with server-defined values.